### PR TITLE
Added Coding Focussed UI

### PR DESCRIPTION
**Pull request by Pavan-Sai-padavala**
1. Added github contributors section at the bottom.
2. Added quote component to show a quote while compiling the code.
3. Applied dark theme

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 const Footer = () => {
   return (
-    <div className="fixed h-8 bottom-0 px-2 py-1 flex items-center justify-center w-full text-xs text-gray-500 ">
+    <div className="h-8 bottom-0 px-2 py-1 flex items-left justify-center w-full text-xs text-gray-200 " style={{fontSize:'large'}}>
```
Comment (manuarora700) [inline src/components/Footer.js:5]: can you use tailwind for this?

File: src/components/Landing.js [hunk 2]
```diff
@@ -262,28 +261,31 @@ const Landing = () => {
           />
         </div>
 
-        <div className="right-container flex flex-shrink-0 w-[30%] flex-col">
-          <OutputWindow outputDetails={outputDetails} />
-          <div className="flex flex-col items-end">
+        <div style={{display:"flex",flexDirection:'row',justifyContent:'space-between'}}>
```
Comment (manuarora700) [inline src/components/Landing.js]: Can you convert inline styles to tailwind?

**Review by manuarora700 — COMMENTED** [review]
Change inline styles to tailwind and we are good to go

**Comment by Pavan-Sai-padavala** [comment]
converted to tailwind... please review it

**Comment by manuarora700** [comment]
> converted to tailwind... please review it
Screenshot 2023-10-07 at 2 13 04 PM
- There's a scroll
- buttons don't match the UI

**Comment by Pavan-Sai-padavala** [comment]
scroll buttons are now matched with the ui, please review the change and merge it
Screenshot (12)

**Pull request closed without merging**